Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -366,6 +366,12 @@ public enum DefaultFunctionOpacityOptions {
 `opaque` means functions are always opaque so the opaque keyword is not needed, and functions must be revealed everywhere needed for a proof.".TrimStart()) {
   };
 
+  public static readonly Option<bool> TranslateStandardLibrary = new("--translate-standard-library", () => true,
+    @"Currently Dafny does not prevent separately built Dafny projects from each including the standard library,
+leading to conflicts. When combining such projects, please ensure that only one of them has --translate-standard-library set to true.
+When building a Dafny library (.doo file), this option will automatically be set to false. 
```

Comment:
This feels like it's going to be confusing and hard to debug, especially since it's a change in behavior. Wouldn't it be better to fail instead, and make users set `--translate-standard-library:false` explicitly?

Reply:
Is there are scenario where you would want to have `--translate-standard-library:true` when using .doo files? I was thinking always excluding them for doo files would strictly be an improvement.

Reply:
True, I was thinking it could be a clean way to get Dafny to produce a doo file equivalent to the internal standard libraries doo file. But you're right that because `--standard-libraries` has to be consistent across doo files the resulting doo file would be unusable.

I'm okay with this change in behavior because the resulting doo file is unusable anyway, so no one can really be relying on this behavior.